test/parallel/test-runner-snapshot-tests.js

```diff
@@ -340,6 +342,30 @@ test('t.assert.snapshot()', async (t) => {
   });
 });
 
+test('special characters are allowed', async (t) => {
+  const fixture = fixtures.path(
+    'test-runner', 'snapshots', 'special-character.js'
+  );
+
+  await common.spawnPromisified(
+    process.execPath,
+    ['--test-update-snapshots', fixture],
```

Comment:
I'm not sure about this approach: updating the test snapshot shouldn't be necessary after it's first created.
This way, it seems we're just verifying that a snapshot file is being created, and I imagine this test would still pass even if we removed the fix.

WDYT? @Ceres6

Reply:
We're actually not asserting anything about the creation process, just that we're able to read from the created file.

If we removed the fix it would fail because the snapshot would be updated with the previous (incorrect) serialization and we wouldn't be able to read it anymore

Reply:
My bad, I missed the second run!  
I'm still a little bit confused about the need to save the snapshot.  
If our goal is only to verify that we can save and read it, is it still needed?

Except for this, it makes sense to me

Reply:
It is not needed, I just didn't know how to avoid saving it! I'll take a look